src/Carbon/Lang/si.php

```diff
@@ -33,15 +33,15 @@
     'a_second' => '{1}තත්පර කිහිපයකට|තත්පර :count',
     'ago' => ':time කට පෙර',
     'from_now' => function ($time) {
-        if (preg_match('/දින \d+/', $time)) {
+        if (preg_match('/දින \d+/u', $time)) {
```

Comment:
Those was converted as is from moment.js but I realize in our case we just want to detect the presence without using the captured number so we can even drop the `+` as we're optimizing it.
```suggestion
        if (preg_match('/දින \d/u', $time)) {
```